goodtablesio/integrations/s3/blueprint.py

```diff
@@ -170,3 +96,204 @@ def create_job():
     _run_validation(bucket, job_id)
 
     return jsonify({'job_id': job_id})
+
+
+# API
+
+# TODO:
+# it should be synced with general
+# approach we use for API (see api blueprint)
+
+
+@s3.route('/api/bucket/<bucket_id>')
```

Comment:
Sorry I didn't catch that previously, but I think we should namespace all api endpoints with the integration name, eg `/api/s3/bucket/<bucket_id>` and `/api/github/repo/<repo_id>`. This will allow us to potentially switch them off, redirect, etc more easily. WDYT?